packages/js/components/src/experimental-select-control/menu.tsx

```diff
@@ -36,7 +59,43 @@ export const Menu = ( {
 				}
 			) }
 		>
-			{ isOpen && children }
-		</ul>
+			<Popover
+				focusOnMount={ false }
+				className={ classnames(
+					'woocommerce-experimental-select-control__popover-menu',
+					{
+						'is-open': isOpen,
+						'has-results': Array.isArray( children )
```

Comment:
I think `'has-results': Array.isArray( children ) && children.length > 0` be a bit clearer. Seems a bit less clear to be setting `has-results` to `children.length` or `Boolean (children)`.

Reply:
Done as part of https://github.com/woocommerce/woocommerce/pull/34967/commits/8e895d4139b8369fd8c90943818bb20c7d10cf06